app/controllers/formbuilder/edit/code.js

```diff
@@ -55,18 +72,16 @@ export default class FormbuilderEditCodeController extends Controller {
 
   async consoleValidateCode(newCode) {
     const shaclReport = await shaclValidateTtlCode(newCode);
-    console.warn(
-      'Formatted SHACL report: ',
-      formatShaclValidationReport(shaclReport)
-    );
-
-    this.warnings = [];
+    this.consoleMessages = A([]);
 
     const formattedReport = formatShaclValidationReport(shaclReport);
+    console.warn('Formatted SHACL report: ', formattedReport);
+
     formattedReport.errorDetails.forEach((error) => {
-      this.warnings.push({
+      this.consoleMessages.pushObject({
+        severity: this.getConsoleSeverity(error.severity),
         subject: error.subject,
-        message: error.messages,
+        content: error.messages,
```

Comment:
As we want to display syntax errors and maybe more this format needs to be as generic as possible. Create a utils object where you can pass data and it returns it as a console message